packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -382,6 +386,21 @@ export default {
 
         updateHasAsapSelected ({ commit }, payload) {
             commit(UPDATE_HAS_ASAP_SELECTED, payload);
+        },
+
+        updateDisplayError: ({ commit }, payload) => {
+            const alert = {
+                code: payload,
+                shouldShowInAlert: true
+            };
+
+            let displayError = null;
+
+            if (payload) {
+                displayError = payload[0].code ? payload[0] : alert;
```

Comment:
Is this correct or should be the other way around: `payload[0] ? payload[0].code : alert`?